document/core/bikeshed/conf.py

```diff
@@ -0,0 +1,470 @@
+# -*- coding: utf-8 -*-
```

Comment:
Nit: move this to /util as well.

Reply:
Moved just the bikeshed one for now.
Probably should consider the other one too at some point. Although it's kind of a gray area whether the config is part of the document or generic.